src/storeinbox.cpp

```diff
@@ -54,15 +54,31 @@ ReturnValue StoreInbox::queryAdd(int32_t, const Thing& thing, uint32_t, uint32_t
 
 void StoreInbox::postAddNotification(Thing* thing, const Cylinder* oldParent, int32_t index, cylinderlink_t)
 {
-	if (parent != nullptr) {
-		parent->postAddNotification(thing, oldParent, index, LINK_PARENT);
+	Cylinder* topParent = getTopParent();
```

Comment:
what's the point of these changes?

Reply:
Since we are 100% sure that the parent of this container will always be the player, then we can reduce the code to how it was before and only change the cylinder link, I think this should be enough, I'm going to try it